docs/docs/basics/resources.md

```diff
@@ -11,7 +11,7 @@ title: 📚 Additional Resources
 - [Livestream demo][livestream_link]
 
 :::info
-Did you know the [dart2js][dart2js_compiler_link] compiler [used to be called frog][dart2js_frog_pr_link]?
+Fun fact: Did you know the [dart2js][dart2js_compiler_link] compiler [used to be called frog][dart2js_frog_pr_link]?
```

Comment:
I thought that adding this might make it clear that this is just for fun. Took me a second when I read the big "Info" banner and "Did you know" to realize this is just fun trivia :)

Reply:
Good call 👍